target/linux/ath79/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -30,6 +30,14 @@ case "$board" in
 		[ "$PHYNBR" -eq 1 ] && \
 			echo $(k2t_get_mac "lan_mac") > /sys${DEVPATH}/macaddress
 		;;
+	trendnet,tew-823dru)
+	        # set the 2.4G interface mac address to LAN MAC
+	        [ "$PHYNBR" -eq 1 ] && \
+		        echo $(mtd_get_mac_text mac 4) > /sys${DEVPATH}/macaddress
+	        # set the 5G interface mac address to WAN MAC + 1
+	        [ "$PHYNBR" -eq 0 ] && \
+		        echo $(macaddr_add "$(mtd_get_mac_text mac 24)" 1) > /sys${DEVPATH}/macaddress
```

Comment:
Same here:
macaddr_add "$(mtd_get_mac_text mac 24)" 1 > /sys${DEVPATH}/macaddress

Reply:
BTW: Since this is also the case for the other entries in this file, I have just send a patch for that to the list ...